app/models/case/overturned_ico/foi.rb

```diff
@@ -1,7 +1,7 @@
 class Case::OverturnedICO::FOI < Case::OverturnedICO::Base
 
   def self.type_abbreviation
-    'OVERTURNED FOI '
+    'OVERTURNED_FOI '
```

Comment:
There's still a space here, I'm guessing this isn't getting in the way because we don't create overturned FOIs? We can fix this later when it becomes an issue.